package.json

```diff
@@ -13,6 +13,19 @@
     "android": "react-native run-android",
     "snyk-protect": "snyk protect"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint . --fix",
+      "eslint .",
+      "jest ./__tests__/*",
+      "git add ."
```

Comment:
I don't think it is a good idea to stage all files all the time

Reply:
Probably you are right, never thought about it and always used this way on all projects. Thanks!

Reply:
But if I remove `eslint . fix` then there is no need to do `git add` so I will remove it anyway